README.md

```diff
@@ -13,7 +13,7 @@
 
 ## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://shella12.github.io/AwesomeBook.github.io/)
 
 
 ## Getting Started
```

Comment:
Great job @shella12 🤝

- [ ] [Optional] Kindly format your readme file accordingly. Add the project description, and project name, and add the correct Github handle to the rightful author.